packages/kbn-i18n/src/core/i18n.ts

```diff
@@ -235,9 +185,16 @@ export async function load(translationsUrl: string) {
     credentials: 'same-origin',
   });
 
-  if (response.status >= 300) {
+  if (response.status >= 400) {
     throw new Error(`Translations request failed with status code: ${response.status}`);
   }
 
-  init(await response.json());
+  const newTranslation = await response.json();
+  if (!newTranslation || !newTranslation.locale || typeof newTranslation.locale !== 'string') {
+    return;
+  }
+
+  await polyfillLocale(normalizeLocale(newTranslation.locale));
```

Comment:
Polyfills were bundled up until now, and now they are fetched `async`.
This means an extra round trip (potentially), after fetching the translations.
Perhaps we could load both translations and polyfills in parallel, with a `Promise.all()` WDYT?